app/components/listing/LocationTimesAccordion.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react";
+import { LocationDetails } from "models";
+import classNames from "classnames";
+import styles from "./LocationTimesAccordion.module.scss";
+
+/*
+    Accordion built per a11y guidelines:
+    https://www.w3.org/WAI/ARIA/apg/patterns/accordion/
+*/
+
+const LocationTimesAccordion = ({
+  locationRenderer,
+  locations,
+}: {
+  locations: LocationDetails[];
+  locationRenderer: (loc: LocationDetails) => React.ReactElement;
+}) => {
+  const [activeIndices, setActiveIndices] = useState<number[]>([0]);
+
+  const toggleAccordion = (index: number) => {
+    setActiveIndices((prevActiveIndices) =>
+      prevActiveIndices.includes(index)
+        ? prevActiveIndices.filter((i) => i !== index)
+        : [...prevActiveIndices, index]
+    );
+  };
+
+  return (
+    <div className={styles.locationTimesAccordion}>
+      {locations.map((loc, i) => {
```

Comment:
Suggestion: Use descriptive names for `location` and `locationIndex` to avoid confusion. I understand `loc` is pretty common shorthand for `location` in a mapping contexts and libraries, but I think it's beneficial for our app code to be as ruthlessly clear as we can within the scope our own domain concepts.